.env.development

```diff
@@ -15,8 +15,9 @@ GITHUB_CLIENT_SECRET=""
 GOOGLE_CLIENT_ID=""
 GOOGLE_CLIENT_SECRET=""
 EMAIL_SERVER=""
-SUPABASE_API_URL=""
```

Comment:
Should we maybe leave the Supabase env vars as an example of the vars needed to use the Supabase client?

Reply:
Of course, I don't know why I removed it